src/tests/ftest/util/server_utils.py

```diff
@@ -303,7 +303,7 @@ class DaosServerManager(SubprocessManager):
         "OFI_PORT": "fabric_iface_port",
     }
 
-    def __init__(self, server_command, manager="Orterun"):
+    def __init__(self, server_command, manager="Orterun", dmg_cfg=None):
```

Comment:
It's better to add description for dmg_cfg.

Reply:
Done.